matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/pushers/DefaultPushersService.kt

```diff
@@ -107,13 +103,6 @@ internal class DefaultPushersService @Inject constructor(
         return request.id
     }
 
-    private fun JsonPusher.validateParameters() {
```

Comment:
as we're always validating the payload I've lifted the logic as close of possible to the fields by doing it in the given class's `init`